client/my-sites/plans/plan-overview/plan-feature/style.scss

```diff
@@ -0,0 +1,9 @@
+.plan-feature__description {
+	em {
```

Comment:
It'd be nice if these were classes. Generic selectors would traverse the DOM looking for all `em`s.